.github/workflows/e2e-suite.yml

```diff
@@ -143,75 +184,10 @@ jobs:
             });
             return result;
 
-  apply-calico-rules:
```

Comment:
These jobs were recently added, let's keep these jobs as it makes the workflow easier to read